test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/capability/CM02/cm02t001/cm02t001.cpp

```diff
@@ -60,6 +60,10 @@ static jlong timeout = 0;
 /* test objects */
 static jthread thread = NULL;
 static jclass klass = NULL;
+static jobject testedObject = NULL;
+const jlong TESTED_TAG_VALUE = (5555555L);
```

Comment:
Remove parenthesis?

Reply:
I copied this from some other place that had parenthesis.